pegasus/test/test_section_api_helpers.rb

```diff
@@ -171,6 +171,38 @@ class SectionApiHelperTest < SequelTestCase
       FakeDashboard.use_fake_database
     end
 
+    describe 'teachers' do
+      it 'returns an array of hashes of information' do
+        pegasus_section = DashboardSection.fetch_if_teacher(
+          FakeDashboard::SECTION_NORMAL[:id],
+          FakeDashboard::TEACHER[:id]
+        )
+        teachers = pegasus_section.teachers
+        assert_equal(
+          [{id: FakeDashboard::TEACHER[:id], location: "/v2/users/#{FakeDashboard::TEACHER[:id]}"}],
+          teachers
+        )
+      end
+    end
+
+    describe 'teacher' do
+      it 'returns false for a non-teacher' do
+        pegasus_section = DashboardSection.fetch_if_teacher(
+          FakeDashboard::SECTION_NORMAL[:id],
+          FakeDashboard::TEACHER[:id]
+        )
+        refute pegasus_section.teacher?(FakeDashboard::TEACHER_SELF[:id])
```

Comment:
If I'm understanding this test correctly, we're passing `pegasus_section.teacher?` a teacher, but one that does not teach this section (and thus we expect it to return false).

Assuming a correct understanding, I might describe this slightly differently, perhaps `it 'returns false for a teacher of a different section'`?